packages/integrations/image/src/lib/get-image.ts

```diff
@@ -107,7 +107,9 @@ export async function getImage(
 
 	if (!loader) {
 		// @ts-ignore
-		const { default: mod } = await import('virtual:image-loader');
+		const { default: mod } = await import('virtual:image-loader').catch((error) => {
```

Comment:
this will actually still fail, since it would return `undefined` and you can't do `const {default: mod} = undefined`.

Created a new PR to solve this along with another error here: https://github.com/withastro/astro/pull/4279

Thanks for helping to point the way to this part of the code!

Reply:
Thanks for your feedback, I saw that you were able to include this part in your PR, glad if I could help you 🙂